### DISCUSSION TITLE
Error: 3002 - Request failed with status code 400:

### DISCUSSION THREAD

**Discussion by janowo2008** [wazuh/wazuh#25281] [Q&A]
After removing logs folders ("archives, alerts) to new partition I have error in WAZUH-DASHBOARD- Error: 3002 - Request failed with status code 400:
"Error: Error: 3002 - Request failed with status code 400
    at WzRequest.genericReq (https://192.168.131.201/48104/bundles/plugin/wazuh/wazuh.plugin.js:1:126595)
    at async WzAuthentication.login (https://192.168.131.201/48104/bundles/plugin/wazuh/wazuh.plugin.js:1:164474)
    at async refresh (https://192.168.131.201/48104/bundles/plugin/wazuh/wazuh.plugin.js:1:164799)"

My Wazuh instalation is "1-Node instalation" with structure Wazuh Indexer (Server-1, IP 192.168.131.200) Wazuh Manger (Server-2, IP 192.168.131.201), Wazuh Dashboard (Server-2, IP 192.168.131.201). Servers Operating Systems are Ubuntu 24.04 LTS Serwer.
https://groups.google.com/g/wazuh/c/tOHFILyvpHc  

![auto_token_fstab_08-20-2024_01](https://github.com/user-attachments/assets/1a73eb52-17ba-4e13-9976-5311c05a235a)

[journalctl--unit_wazuh-dashboard.service.txt](https://github.com/user-attachments/files/16674298/journalctl--unit_wazuh-dashboard.service.txt)

![nano_fstab_08-20-2024_01](https://github.com/user-attachments/assets/ad91676c-f301-4809-822f-45a957bcbcde)

**Comment by zlsq** [comment]
same error， did you solve this problem?

**Reply by Crazygiscool** [reply]
me too, can't get a solve, I believe it is because the SSL is self signed, but idk
